src/screens/SUE/SueReportScreen.tsx

```diff
@@ -211,9 +211,21 @@ export const SueReportScreen = ({
         }
         break;
       case 2:
-        if (getValues().zipCode?.length && getValues().zipCode.length !== 5) {
+        if (getValues().houseNumber && !getValues().street) {
+          return texts.sue.report.alerts.street;
+        }
+
+        if (!getValues().zipCode) {
+          return texts.sue.report.alerts.zipCode;
+        }
+
+        if (getValues().zipCode.length !== 5) {
           return texts.sue.report.alerts.zipCodeLength;
         }
+
+        if (!getValues().city) {
+          return texts.sue.report.alerts.city;
+        }
```

Comment:
we should add `*` to these two fields, right?

Reply:
Yes 👍
 https://github.com/smart-village-solutions/smart-village-app-app/commit/381cfc039e76f6793924224c2df3cbb1447a7aa1